### ISSUE TITLE
`isgen` not defined

### ISSUE THREAD

**Issue by SoongNoonien** [oscar-system/GenericCharacterTables.jl#32]
In `speccharparam!` and `specclassparam!` we use `isgen` to determine if a universal polynomial is a plain indeterminate of its parent ring. Apparently `isgen` doesn't exist anymore.

@fingolfin Has this been renamed during the great renaming of Oscar?

**Comment by fingolfin** [comment]
Yes. It was changed to match our general rule and is now called `is_gen` (and that has been the default, non-deprecated version since May 2022 :-) )
